services/player-service/infrastructure/src/test/resources/application-test.yml

```diff
@@ -1,14 +1,15 @@
 server:
-  port: ${SERVER_PORT}
+  port: 8082
```

Comment:
you don't need to override. It will come from default application.yml.

Reply:
Removed!